source/material_model/rheology/strain_dependent.cc

```diff
@@ -88,7 +88,12 @@ namespace aspect
                            "but is there to make sure that the original parameters for specifying the "
                            "strain weakening mechanism (``Use plastic/viscous strain weakening'') are still allowed, "
                            "but to guarantee that one uses either the old parameter names or the new ones, "
-                           "never both.");
+                           "never both."
+                           "\n\n"
+                           "If a compositional field named noninitial\\_plastic strain is "
```

Comment:
the field name has an underscore in the code. Is this a mistake here? Can you also put the name in quotes please?

Reply:
There should have another underscore to give `noninitial_plastic_strain`. Thanks for catching that. The name is also in quotes now as well.